### PR TITLE
Drop support for EOL Python 3.4

### PR DESCRIPTION
**Pull request by hugovk**
Python 3.4 is EOL since 2019-03-16 and no longer receiving security updates (or any updates) from the core Python team.

Source: https://en.wikipedia.org/wiki/CPython#Version_history

It's also little used.

Here's the pip installs for pyscreenshot from PyPI for April 2019:

| category | percent | downloads |
|----------|--------:|----------:|
|      2.7 |  69.20% |    27,363 |
|      3.6 |  14.05% |     5,557 |
|      3.7 |  13.93% |     5,509 |
|      3.5 |   1.77% |       698 |
| null     |   0.79% |       312 |
|      3.4 |   0.26% |       101 |
|      3.8 |   0.01% |         3 |
| Total    |         |    39,543 |

Source: `pypistats python_minor pyscreenshot --last-month # pip install pypistats`

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -26,12 +26,12 @@ Features:
  * saving to PIL_ or Pillow_ image memory
  * some back-ends are based on this discussion: http://stackoverflow.com/questions/69645/take-a-screenshot-via-a-python-script-linux
  * pure Python library
- * supported Python versions: 2.7, 3.4, 3.5, 3.6, 3.7
+ * supported Python versions: 2.7 and 3.5+
```
Comment (ponty) [inline README.rst]: I prefer explicit versions: 3.5, 3.6, 3.7

Comment (hugovk) [reply README.rst]: My thinking was that it's one less line to update when 3.8 is released, but would you like a new PR to add back explicit versions?

**Comment by ponty** [comment]
Thanks

File: README.rst (hunk 1 above)
Comment (hugovk) [reply README.rst]: I see you've taken care of it in https://github.com/ponty/pyscreenshot/commit/f78816044e5555b7b62abdf4450b022cc7cf8ed9. 👍

**Pull request merged**